contracts/JBETHPaymentTerminalStore.sol

```diff
@@ -403,8 +400,7 @@ contract JBETHPaymentTerminalStore {
     }
 
     // Convert the amount to wei.
-    // A currency of 0 should be interpreted as whatever the currency being distributed is.
```

Comment:
This seemed a bit tribal to me so I generalized the logic to skip if the preferred currency is ETH; otherwise we do the price conversion.

Reply:
nice